CredentialCardDetail.vue

```diff
@@ -31,6 +31,14 @@
             {{issuerName}}
           </q-item-section>
         </q-item-label>
+        <q-chip
```

Comment:
This should not exist in this component. Credential Cards should be dumb components, they have no notion of a "Profile". Credential cards are just a visual representation of a Verifiable Credential.

Reply:
We need to show the profile that the card is associated with in the credentials details so the user can understand what profile the credential is under.